ChangeLog.md

```diff
@@ -1,11 +1,115 @@
+   Windows conventions.  "/dev/null" remains correct on Linux. (This feature can be used to test
+   throughput or check MD5s without saving the downloaded data.) 
+1. The file format of the (still undocmented) `--list-of-files` parameter is changed.  (It remains
+   undocmented because, for simplicity, users are
+   encouraged to use the new `--include-pattern` and `--include-path` parameters instead.)
+
+### New features
+
+1. `sync` is supported from Blob Storage to Blob Storage, and from Azure Files to Azure Files.
+1. `copy` is supported from Azure Files to Azure Files, and (TBC?) from Blob Storage to Azure Files.
+1. Percent complete is displayed as each job runs.
+1. VHD files are auto-detected as page blobs.
+1. A new benchmark mode allows quick and easy performance benchmarking of your network connection to
+   Blob Storage. Run AzCopy with the paramaters `bench --help` for details.  This feature is in
+   Preview status.
+1. The location for AzCopy's "plan" files can be specified with the environment variable
+   `AZCOPY_JOB_PLAN_LOCATION`.
+1. Log files and plan files can be cleaned up to save disk space, using AzCopy's new `jobs rm` and
+   `jobs clean` commands.
+1. When listing jobs with `jobs show`, the status of each job is included in the output.   
+1. The `--overwrite` parameter now supports the value of "prompt" to prompt the user on a
+   file-by-file basis. (The old values of true and false are also supported.)   
+1. The environment variable `AZCOPY_CONCURRENCY_VALUE` can now be set to "AUTO". This is expected to be
+    useful for customers with small networks, or those running AzCopy on
+   moderately-powered machines and transfer blobs between accounts.  This feature is in preview status.
+1. When uploading from Windows, files can be filtered by Windows-specific file attributes (such as
+   "Archive", "Hidden" etc)
+1. Memory usange can be controlled by setting the new environment variable `AZCOPY_BUFFER_GB`.
+   Decimal values are supported. Actual usage will be the value specified, plus some overhead. 
+1. An extra integrity check has been added for Service to Service transfers: the length of the
+   completed desination file is checked against that of the source.
+1. When downloading, AzCopy can automatically decompress blobs (or Azure Files) that have a
+   `Content-Encoding` of `gzip` or `deflate`. To enable this behaviour, supply the `--decompress`
+   parameter.
+1. The number of disk files accessed concurrently can be controlled with the new
+   `AZCOPY_CONCURRENT_FILES` environment variable. This is an advanced setting, which generally
+   should not be modified.  It does not affect the number of HTTP connections, which is still
+   controlled by `AZCOPY_CONCURRENCY_VALUE`.
+1. The values of key environment variables are listed at the start of the log file.
+1. An official 32-bit build is now released, in addition to the usual 64-bit builds. (TBC: For which OS's?)
+
+### Bug fixes
```

Comment:
This feels like a fitting place to put this, but as a part of the "log some more info" PR:

- S2S access tier preservation failures are logged
- Container name resolution is only checked on scheduled containers when using a wildcard, rather than ALL containers in an account
    - I'm starting to think this one was intentional, regarding our earlier discussion of why the name resolution was done ahead of time, actually. I don't think it's entirely something we need to worry about, however. Consider the following: A user schedules a transfer copying bucket names bucket*name, which targets `bucket-name` and `bucket.name`. `bucket.name` becomes `bucket-name-2`. Then, consider the user (somehow) scheduled a transfer which included *just* `bucket.name`. `bucket.name` resolves to `bucket-name`, and the files go to the wrong place.
    - I don't think this is so severe, though, because the same scenario would happen if `bucket-name` was deleted. There's no _possible_ way we could expect that unless we stored the map of bucket names (which I already feel is a poor idea in its own sense)

Reply:
I think the container issue is covered by a backlog item. Search for "Make S2S bucket name resolution more robust from S3"